pyproject.toml

```diff
@@ -11,6 +11,7 @@ PyYAML = "^6.0"
 gatorgrader = "^1.1.1"
 rich = "^12.5.1"
 typer = {extras = ["all"], version = "^0.6.1"}
+urllib3 = "<=2.0"
```

Comment:
Do we need to add this dependency explicitly, or is this to ensure we resolve a version lower than 2.0? If it is, we need to use `<`, not `<=`. I'd prefer letting transitive dependencies resolve automatically if possible.

Reply:
Oh, it's not necessary and won't work. It was a potential solution I tried and failed. I'd like to delete this change in `pyproject.toml`